app/controllers/AMIable.scala

```diff
@@ -56,12 +56,15 @@ class AMIable @Inject()(override val amiableConfigProvider: AmiableConfigProvide
           totalInstancesCount = instances.length,
           PrismLogic.instancesAmisAgePercentiles(instancesWithAmis)
         )
+        allSSAs = ssa :: PrismLogic.instanceSSAs(instances)
```

Comment:
Why do we need to add the requested SSA to the list? `instances` came from searching for that AMI so either it's already there, or there aren't any instances with the ssa.